app/views/homepage/_promotion-slots.html.erb

```diff
@@ -2,119 +2,55 @@
   Before updating a promotion slot please read the documentation.
   See: https://docs.publishing.service.gov.uk/apps/frontend/update-homepage-promotion-slots.html
 %>
-<section>
+<section class="homepage-section">
+  <div class="homepage-section__heading">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: t('homepage.index.featured')
```

Comment:
Minor formatting change - [double quotes should be used where possible](https://github.com/alphagov/rubocop-govuk/blob/ce655779e45892db6ac00d6789c652ec2e506748/config/style.yml#L47-L53):
```suggestion
      text: t("homepage.index.featured")
```